packages/ibm-products/src/components/UserAvatar/UserAvatar.test.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright IBM Corp. 2024, 2024
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import { render, screen } from '@testing-library/react'; // https://testing-library.com/docs/react-testing-library/intro
+
+import { pkg } from '../../settings';
+import uuidv4 from '../../global/js/utils/uuidv4';
+
+import { UserAvatar } from '.';
+
+const blockClass = `${pkg.prefix}--user-avatar`;
+const componentName = UserAvatar.displayName;
+
+// values to use
+const dataTestId = uuidv4();
+
+const renderComponent = ({ ...rest } = {}) =>
+  render(<UserAvatar {...{ ...rest }} />);
+
+describe(componentName, () => {
+  it('should return a circle with background color', async () => {
+    const { container } = renderComponent({
+      backgroundColor: 'light-cyan',
+    });
+    const element = container.querySelector(`.${blockClass}`);
```

Comment:
If possible avoid `querySelector`, this can be something refactored as you work more on this component.

Reply:
Done the required changes.